linux_os/guide/system/auditing/auditd_configure_rules/audit_unsuccessful_file_modification/audit_rules_unsuccessful_file_modification_open_o_creat/oval/shared.xml

```diff
@@ -0,0 +1,200 @@
+<def-group>
+  <definition class="compliance" id="audit_rules_unsuccessful_file_modification_open_o_creat" version="1">
+    <metadata>
+      <title>Ensure auditd Collects Information on Unsuccesful Creation Attemps to Files</title>
+      <affected family="unix">
+        <platform>Red Hat Enterprise Linux 7</platform>
+        <platform>multi_platform_fedora</platform>
+      </affected>
+      <description>Audit rules about the information on the use of privileged commands are enabled.</description>
```

Comment:
<description>Audit rules about the information on the unsuccessful use of O_CREAT is enabled.</description>